packages/beacon-node/src/api/impl/beacon/pool/index.ts

```diff
@@ -104,7 +104,11 @@ export function getBeaconPoolApi({
           try {
             await validateBlsToExecutionChange(chain, blsToExecutionChange);
             chain.opPool.insertBlsToExecutionChange(blsToExecutionChange);
-            await network.gossip.publishBlsToExecutionChange(blsToExecutionChange);
+            try {
+              await network.gossip.publishBlsToExecutionChange(blsToExecutionChange);
+            } catch (e) {
+              await chain.cacheBlsToExecutionChanges(blsToExecutionChange);
```

Comment:
Swallowing errors generically is very ugly, why not do
```
if (after_capella)
  publish
else
  persist
```

Reply:
i was trying to go for handling any persitent errors, but this looks simpler in terms of UX, so will modify :+1: